python-packages/securesync/devices/models.py

```diff
@@ -207,10 +207,15 @@ def set_key(self, key):
         self.key = key
 
     def get_key(self):
+        self.key = None
         if not self.key:
```

Comment:
Oops. Actually, I broke the caching. Should use the old `self.key` if it exists.